topology.yaml

```diff
@@ -0,0 +1,32 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# yaml definition file for the nodebase Kerberos topology.
+
+name: nodebase_kerberos
+description: A basic Kerberos (KDC) topology
+
+node groups:
+    kdc-node:
+        - kdc
+    nodes:
+        - node-1
+        - node-2
+
+start args:
+    --kerberos-config-directory:
+        default: ~/.clusterdock/kerberos
+        help: If specified, mounts this directory to KDC container for Kerberos config files.
```

Comment:
Remove `If specified, ` from help message since it has a default.

Reply:
Done